packages/sdk/src/getTransferData/getDestinationData.ts

```diff
@@ -51,12 +56,18 @@ export async function getDestinationData({
     polkadot,
   });
   const minAmount = zeroAmount.copyWith({ amount: min });
+
   return {
     balance: balanceAmount,
     chain,
     existentialDeposit,
     fee: feeAmount,
     min: minAmount,
+    sovereignAccountBalances: await getSovereignAccountBalances({
```

Comment:
As I understand, it will always get the Sovereign account balance but is it needed? Maybe we have to put something to the config. For example a flag. If it's true then we get the balance

Reply:
I was not sure if adding a flag or not. But yeah maybe it's better to do that and remove the check by chain in the validation function. Will also prevent potential problems getting the sovereign account balances in other chains that we haven't tested